packages/clarity-js/src/layout/mutation.ts

```diff
@@ -81,54 +81,55 @@ export function end(): void {
 }
 
 function handle(m: MutationRecord[]): void {
+  console.log(time());
   // Queue up mutation records for asynchronous processing
-  mutations.push({ time: time(), mutations: m});
+  mutations.push({ time: time(), mutations: m });
   task.schedule(process, Priority.High).then((): void => {
-      measure(doc.compute)();
-      measure(region.compute)();
+    measure(doc.compute)();
```

Comment:
Nitpick: We should get our IDE in sync so we don't have to look through these weird space differences.